assets/js/blocks/cart-checkout/cart/full-cart/cart-line-items-table.tsx

```diff
@@ -29,6 +29,20 @@ const CartLineItemsTable = ( {
 					<CartLineItemRow
 						key={ lineItem.key }
 						lineItem={ lineItem }
+						onRemove={ ( removedRow: HTMLElement | null ) => {
+							if (
+								removedRow?.nextElementSibling instanceof
+								HTMLElement
+							) {
+								removedRow.nextElementSibling.focus();
+							} else if (
+								removedRow?.previousElementSibling instanceof
+								HTMLElement
+							) {
+								removedRow.previousElementSibling.focus();
+							}
+						} }
+						tabIndex={ -1 }
```

Comment:
why is this tabIndex neeed?

Reply:
Preferably  `tabIndex={ -1 }` should be avoided. The tabbed navigation in the mini cart worked really well before this PR when navigating from one item the other  after deleting. 

I think removing items from the normal flow is the reason why the issues from https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4673#issuecomment-912623764 happen. 

The only issue befre this PR was focusing after removing the last item. I am thinking it would be easier just to move the focus to the mini cart wrapper, and not try to focus the previous item. What do you think?